src/dms/core/EdFi.DataManagementService.Core/Security/AuthorizationValidation/RelationshipsWithEdOrgsOnlyValidator.cs

```diff
@@ -37,7 +37,11 @@ OperationType operationType
         }
 
         // Retrieve the hierarchy of the education organization ids from the request
-        var edOrgIdValues = edOrgsFromRequest.Select(e => e.Value).ToArray();
+        var edOrgIdValues = edOrgsFromRequest
```

Comment:
They should already be strings at this point.

Reply:
I updated it . Thanks !